tests/test_bonded.py

```diff
@@ -389,7 +389,7 @@ def test_chiral_atom_restraint(self, n_particles=64, n_restraints=35, dim=3):
         for precision, rtol in relative_tolerance_at_precision.items():
             self.compare_forces(x, params, box, potential, potential.to_gpu(precision), rtol)
 
-            with self.assertRaises(Exception):
+            with self.assertRaises(ValueError):
```

Comment:
(ValueErrors are raised by the wrapper code).

Reply:
Where is this raised exactly, don't see any ValueError being raised explicitly?

Reply:
For posterity: this was raised in the `__post_init__()` hooks